### PR TITLE
feat: adds pipx plugin for asdf

### PR DESCRIPTION
**Pull request by joe733**
## Summary

Description:

- Tool repo URL: https://github.com/pypa/pipx
- Plugin repo URL: https://github.com/joe733/asdf-pipx

## Checklist

- [X] Format with `scripts/format.bash`
- [X] Test locally with `scripts/test_plugin.bash --file plugins/pipx`
- [X] Your plugin CI tests are green
  > - _Tip: use the `plugin_test` action from [asdf-actions](https://github.com/asdf-vm/actions) in your plugin CI_

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -482,6 +482,7 @@ The `asdf` core provides a [security policy](https://github.com/asdf-vm/asdf/sec
...
 | pint                          | [sam-burrell/asdf-pint](https://github.com/sam-burrell/asdf-pint)                                               | [![Build Status](https://github.com/sam-burrell/asdf-pint/workflows/ci/badge.svg?branch=main)](https://github.com/sam-burrell/asdf-pint/actions/workflows/ci.yml)                                                                 |
 | pipectl                       | [pipe-cd/asdf-pipectl](https://github.com/pipe-cd/asdf-pipectl)                                                 | [![Build Status](https://github.com/pipe-cd/asdf-pipectl/actions/workflows/build.yml/badge.svg)](https://github.com/pipe-cd/asdf-pipectl/actions)                                                                                 |
+| pipx                          | [joe733/asdf-pipx](https://github.com/joe733/asdf-pipx)                                                         | [![Build Status](https://github.com/pipe-cd/asdf-pipectl/actions/workflows/build.yml/badge.svg)](https://github.com/pipe-cd/asdf-pipectl/actions)                                                                                 |
```
Comment (jthegedus) [inline README.md:485]: The badges need to be updated to point to your plugins repo

Comment (joe733) [reply README.md]: Fixed.

**Comment by joe733** [comment]
@jthegedus anything amiss?

**Comment by joe733** [comment]
Ready.

> The hiccup was: https://github.com/pypa/pipx/issues/974. Resolved by: https://github.com/pypa/pipx/issues/278#issuecomment-554679349

**Pull request merged**